apps/docs/src/app/core/component-docs/content-density-2/content-density-user/content-density-user.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Inject, ViewEncapsulation } from '@angular/core';
+import { DestroyedService } from '@fundamental-ngx/core/utils';
+import {
+    ContentDensityConsumer,
+    contentDensityConsumer,
+    ContentDensityMode
+} from '@fundamental-ngx/core/content-density';
+
+@Component({
+    selector: 'fundamental-ngx-content-density-user',
```

Comment:
Please use `fd-docs-.......`  so it's clear for the user that this is a documentation related component.